### PR TITLE
feat (pimpinan-jabar): add regional secretary section

### PR DESCRIPTION
**Pull request by naufalihsank**
### Related
- [S10A10 - Update Design dan Konten Pimpinan Jabar](https://app.clickup.com/t/2z4pwek)
### Preview
**Mobile**
profil mobile
**Tablet**
profil tablet
**Desktop**
profil desktop
### Evidence
title: add regional secretary section
project: Portal Jabar
participants: @naufalihsank @Ibwedagama @maulanayuseph @agunghide @yoslie

### CODE REVIEW

**Review by Ibwedagama — APPROVED** [review]

File: pages/tentang-jawa-barat/index/pimpinan-jabar.vue [hunk 1]
```diff
@@ -188,70 +227,103 @@ export default {
...
+          name: 'Dr. (H.C.) H. Mochamad Ridwan Kamil, ST., MUD.',
           dateOfBirth: 'Bandung, 4 Oktober 1971',
           educations: [
             {
               id: 1,
-              description: 'S2, Master of Urban Design University of California, Berkeley, AS (1999-2001)'
+              description: 'Program Magister (S2), Master of Urban Design, UC Berkeley, USA (1999-2001)'
             },
             {
               id: 2,
```
Comment (Ibwedagama) [inline pages/tentang-jawa-barat/index/pimpinan-jabar.vue:238]: minor/discussion:

should we move this data to `static/data` @naufalihsank ?

Comment (naufalihsank) [reply pages/tentang-jawa-barat/index/pimpinan-jabar.vue:238]: if we'll continue to store the data in frontend, that would be great

File: pages/tentang-jawa-barat/index/pimpinan-jabar.vue [hunk 2]
```diff
@@ -171,11 +178,43 @@
...
           </ul>
+          <h4 class="font-roboto font-medium text-base text-gray-800 leading-[26px]">
+            LHKPN (Laporan Harta Kekayaan Penyelenggara Negara)
+          </h4>
+          <Link :link="''" :title="''" download class="mb-6 mt-3">
+            <Button
+              variant="secondary"
+              type="button"
+              class="w-full !justify-center"
+              var
```
Comment (agunghide) [inline pages/tentang-jawa-barat/index/pimpinan-jabar.vue:194]: looks like forgot to delete this `var`

File: pages/tentang-jawa-barat/index/pimpinan-jabar.vue (hunk 1 above)
Comment (Ibwedagama) [reply pages/tentang-jawa-barat/index/pimpinan-jabar.vue:238]: Also separating the component would be nice, right now this page looks a little too big 🤔

**Review by yoslie — APPROVED** [review]

File: pages/tentang-jawa-barat/index/pimpinan-jabar.vue (hunk 2 above)
Comment (naufalihsank) [reply pages/tentang-jawa-barat/index/pimpinan-jabar.vue]: whoops, my bad.

fixed [here](https://github.com/jabardigitalservice/portal-jabar-frontend/pull/338/commits/1815952689ba2649a45b353239851a26915b4d06)

thanks!

File: pages/tentang-jawa-barat/index/pimpinan-jabar.vue (hunk 1 above)
Comment (naufalihsank) [reply pages/tentang-jawa-barat/index/pimpinan-jabar.vue:237]: > > if we'll continue to store the data in frontend, that would be great
> 
> Maybe on the next PR 🤣

sure wkwkwk

**Review by maulanayuseph — APPROVED** [review]

File: pages/tentang-jawa-barat/index/pimpinan-jabar.vue (hunk 1 above)
Comment (naufalihsank) [reply pages/tentang-jawa-barat/index/pimpinan-jabar.vue:237]: > Also separating the component would be nice, right now this page looks a little too big 🤔

right, i thought that the profile card already an independent component at first 🤣

**Review by agunghide — APPROVED** [review]

**Pull request merged**